test/processors/ide/idea_launch_processor_test.dart

```diff
@@ -51,11 +51,11 @@ void main() {
 
   test('Test VSCodeLaunchProcessor', () {
     String matcher = File(
-            'test_resources/ide/idea_launcher_processor_test/example.xml')
+            'test_resources/ide/idea_launcher_processor_test/apple.xml')
```

Comment:
`config: flavorizr`didn't contain a flavor named 'example' and so this test seems not valid. Renamed to apple.